spk/homeassistant/src/dsm-control.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python3"
+PYTHONPATH="${INSTALL_DIR}/share/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+PYTHON="${INSTALL_DIR}/env/bin/python3"
+
+CONFIG_DIR="${INSTALL_DIR}/etc"
```

Comment:
I did this because:
- I wanted separation of (config and runtime files) and the pid file.
- Home Assisant writes all config and runtime to the folder specified with -c
- PYTHONPATH is needed to let python find the homeassistant module